apps/site/tests/shared/wrapped-playwright.ts

```diff
@@ -32,7 +32,7 @@ const tolerableSharedConsoleMessageMatches: RegExp[] = [
 // Messages that only show up in `yarn dev` (not CI)
 if (!process.env.CI) {
   tolerableSharedConsoleMessageMatches.push(
-    /Image with src "\/_next\/static\/media\/primary-helix-min\.\w+\.png" was detected as the Largest Contentful Paint \(LCP\)\. Please add the "priority" property if this image is above the fold\./, // https://nextjs.org/docs/api-reference/next/legacy/image#priority
+    /Image with src "\/_next\/static\/media\/.*" was detected as the Largest Contentful Paint \(LCP\)\. Please add the "priority" property if this image is above the fold\./, // https://nextjs.org/docs/api-reference/next/legacy/image#priority
```

Comment:
Fixes `"[warning] Image with src \"/_next/static/media/helix-boxes.33d40be1.webp\" was detected as the Largest Contentful Paint (LCP). Please add the \"priority\" property if this image is above the fold.
+ Read more: https://nextjs.org/docs/api-reference/next/legacy/image#priority",`

<img width="1015" alt="Screenshot 2023-02-27 at 01 18 28" src="https://user-images.githubusercontent.com/608862/221450669-7c5ac05c-567e-4abf-bdf1-d015dd5840bd.png">